File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Prompt intenger from user
    int height = get_int("Height: ");
    // Checks wether height is between 1 and 8
    while (height < 1 || height > 8)
    {
        height = get_int("Height: ");
    }
    
    // Iterate over each line
    for (int i = 0; i < height; i++)
    {
        // Print espaces
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }
        // Print left hahshes
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        // Print the gap
        printf("  ");
        // Print right hashes
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        // Return
        printf("\n");
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

# Main function


def main():
    n = get_height()
    for i in range(n):
        print(" " * (n - i - 1) + "#" * (i + 1) + "  " + "#" * (i + 1))
    
# Prompt the user for an integer between 1 and 8


def get_height():
    n = get_int("Height:")
    while n < 1 or n > 8:
        n = get_int("Height:")
    return n
    
    
# Call the main function
main()